### DISCUSSION TITLE
Orchard Bounty Feature

### DISCUSSION THREAD

**Discussion by fuzl-llc** [OrchardCMS/OrchardCore#14965] [Ideas]
Maybe this already exists somewhere but I haven't been able to find it... I'd love to see a "bounty" feature added somehow, maybe even just informally as a new type of discussion here. I've gotten lots of great help here, and many times it is in our best interest to learn enough about Orchard to do what we want. However, sometimes something would require a ton of time and we really just need it done. 

For example, right now I would like to re-use the very cool drag/drop hierarchical editor for the Menu content type so I can create a widget that lets me edit hierarchies of content items. I started looking through Orchard's menu module and it was pretty overwhelming, so I'm just giving up for now. However, I bet there are some people who could knock out in an hour what might take me ten times as long to do. I'd love to be able to post a challenge and pay for someone to solve it. We could even post the code somewhere for others to learn from.

Does anything like this exist or would it be of interest to anyone?

**Comment by Piedone** [comment]
There's no formal way of doing something like this, but it's not uncommon that community members contribute fixes/improvements as part of their jobs or by hire ([these PRs](https://github.com/OrchardCMS/OrchardCore/pulls?q=is%3Apr+is%3Aopen+lombiq+technologies) are like that too).

I wouldn't necessarily bring monetary incentives into the discussion board officially, but you may indicate in your question that you're willing to pay for help.
